src/app/dataValidator/rules/topstories.test.js

```diff
@@ -0,0 +1,134 @@
+import applySquashTopstories from './topstories';
+
+// data with the groups 'top-story' 'secondary-top-story' 'other-top-stories'
+import fixtureDataFormatOne from '../../../../data/prod/thai/frontpage/index.json';
```

Comment:
Should we specify what the two formats are/how they differ? We might know why we have two types of data feeds, but others might not

Reply:
Done :), moved comment to the bottom where the `describe` is